frontend/src/Judge/components/PhotoSubmission.js

```diff
@@ -1,5 +1,57 @@
-import React from 'react'
+import React, { Component, Fragment } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Lightbox from 'react-images'
 
-const PhotoSubmission = props => <div>Photo Submission</div>
+const PhotoContainer = styled.img`
+  height: 100%;
+  object-fit: scale-down;
+  width: 100%;
+`
+
+class PhotoSubmission extends Component {
+  static propTypes = {
+    path: PropTypes.string.isRequired
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      lightboxIsOpen: false
+    }
+  }
+
+  closeLightbox = () => {
+    this.setState({ lightboxIsOpen: false })
+  }
+
+  openLightbox = () => {
+    this.setState({ lightboxIsOpen: true })
+  }
+
+  render () {
+    const { path } = this.props
+
+    return (
+      <Fragment>
```

Comment:
This seems to be duplicated here and in the `OtherMediaSubmission` component. Consider creating a component which is shared between them.

Reply:
`<PhotoSubmission path={path}` used in OtherMedia